ParisBNF/abb/BNFabb142.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb142" xml:lang="en" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Prayer of Our Lady Mary on Mount Golgotha, Prayer against colic</title>
+            <editor role="cataloguer" key="DN"/>
+            <editor role="generalEditor" key="AB"/>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+               Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </p>
+               </licence>
+            </availability>
+
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+              <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                    <idno>Éthiopien d'Abbadie 142</idno>
+                    </msIdentifier>
+               <msContents>
+                  <summary>
+                  </summary>
+
+                  <msItem xml:id="ms_i1">
+                    <locus from="1r"/>
+                     <title type="complete" ref="LIT2246salotz"/>
+                     <textLang mainLang="gez"/>
+                      </msItem>
+
+                           <msItem xml:id="ms_i2">
```

Comment:
this is a1 (GuestText), not msItem
![image](https://user-images.githubusercontent.com/17987728/215472675-c041a446-18b8-4b27-a2c3-e7b20f485b85.png)

Reply:
It can be, but Conti Rossini catalogued it as the second item, not as additional note (perhaps the hand is the same, the combination of texts is not impossible). I just followed him. This manuscript is not on Gallica, I would leave it now as it is.

Reply:
I did see it but if this is a guard leaf then it cannot be msItem